dev/com.ibm.ws.jaxrs.2.x.webcontainer_fat/fat/src/com/ibm/ws/jaxrs2x/fat/helloworld/HelloWorldTest.java

```diff
@@ -0,0 +1,176 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.jaxrs2x.fat.helloworld;
+
+/*
+ * IBM Confidential
+ *
+ * OCO Source Materials
+ *
+ * Copyright IBM Corp. 2011
+ *
+ * The source code for this program is not published or other-
+ * wise divested of its trade secrets, irrespective of what has
+ * been deposited with the U.S. Copyright Office.
+ */
+import java.io.File;
+
+import static org.junit.Assert.fail;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class HelloWorldTest {
+    @Server("com.ibm.ws.jaxrs2x.fat")
+    public static LibertyServer server;
+
+//    private final Class<?> c = HelloWorldTest.class;
```

Comment:
```suggestion
```

Let's go ahead and remove any dead code.

Reply:
Done